sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/utils.py

```diff
@@ -590,6 +641,18 @@ def reformat_agent_response(response, logger = None):
             logger.warning(f"Agent response could not be parsed, falling back to original response: {response}")
         return response
 
+
+def reformat_tool_definitions(tool_definitions, logger=None):
+    output_lines = ["TOOL_DEFINITIONS:"]
+    for tool in tool_definitions:
+        name = tool.get("name", "unnamed_tool")
```

Comment:
Not all tools may follow this format, please validate if this works for non function type tools